### ISSUE TITLE
Another busted solution with VS 2015. Please update your blog and associated .NETCore solutions on GIT, or remove them. Thank you.

### ISSUE THREAD

**Issue by crosehaascnc** [tonysneed/aspnet-webapi-2-contrib#3]
Old content: this, and all associated Solutions need to be reworked for VS2015 Update 3. Thank you.

**Comment by tonysneed** [comment]
@crosehaascnc, Can you please be more specific?  Are you talking about the samples for this repo?  Are you getting some kind of error?

**Comment by crosehaascnc** [comment]
Unable to build the downloaded Solution with Visual Studio 2015 Update 3. The Solutions posted on GIT reference earlier incarnations of .NETCore; therefore do not build with Visual Studio 2015 Update 3. Things have changed…

Cheers.

**Comment by tonysneed** [comment]
@crosehaascnc I think your issue doesn't have anything to do with this repo, which has nothing to do with .NET Core.  This repo only pertains to Web API 2, which is .NET 4.x, not .NET Core.

If you are using .NET Core, then the functionality you seek is now built into the product.  In fact, I submitted a PR for it, and what we did was add a SerializerSettings property to the XML serializer, to allow you to configure it to handle cycles.  So the MVC 6 helpers repo was taken down.

I will update the blog post with this info.
